Trumpf.Coparoo.Web/Controls/Table/TableHeadCell.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Trumpf.Coparoo.Web.Controls.Table
```

Comment:
Remove class once it is replaced by the cell class from above.
Ideally, the cell class has public properies `IsHeaderCell` and `IsBodyCell` of type `bool` that return true or false, depending on the tag of the `.Node`-element.